docs/index.md

```diff
@@ -3,9 +3,9 @@
 
 Installation is as simple as:
 
-```bash
+````bash
```

Comment:
I've rendered the docs locally based on your changes but I couldn't see any change in rendered docs.

What should we expect by this change?

Reply:
Great question. The reason for this change is to fix line numbering around the code fences.

Reply:
I will remove in next commit.

Reply:
You've resolved this conversation but I still can see four back quotes in your change.
I think you need to revert all of them